content/reference/sql/server/debugging.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Debugging
+---
+
+# Debugging
+
+Debugging a running Dolt server can be challenging. This document covers the debugging basics and how to diagnose what is happening from common symptoms.
+
+# Basics
+
+## Make sure you are running the latest Dolt version
+
+Dolt is constantly evolving. We release a new Dolt approximately once a week. Connect to the server and run `dolt version`. Make sure the version matches the latest as seen on the [GitHub releases page](https://github.com/dolthub/dolt/releases). 
+
+To upgrade the server, download the latest Dolt binary for your platform, replace the current Dolt binary with the current one, and restart the Dolt server using `dolt sql-server`.
+
+## Examine your CPU, Memory, and Disk usage
+
+Dolt consumes CPU, Memory, and Disk. Consuming more of any of these resources than the host has available can lead to degraded performance. Use your system's built in resource monitoring systems to inspect Dolt's usage of these resources. You may need a larger host or additional [read replicas](./replication.md) to support your load.
```

Comment:
I think we don't know the answer for this, but... reading this made me wonder what I should expect cpu/memory/disk usage to be. It would be awesome if we could help customers understand what to expect, but I understand that's probably not v0 of this doc. :-)